plugins/leemons-plugin-academic-calendar/frontend/src/hooks/useListRegionalConfigs.js

```diff
@@ -0,0 +1,11 @@
+import { useQuery } from '@tanstack/react-query';
+import { listRegionalConfigs } from '../request/regional-config';
+
+export const useListRegionalConfigs = (center, options = {}) =>
+  useQuery({
+    queryKey: ['listRegionalConfigs', center],
+    queryFn: () => listRegionalConfigs(center),
+    ...options,
+  });
```

Comment:
React query key does not match the defined standard

Reply:
implement queryKey standard